src/SprykerEco/Zed/Episerver/Business/EpiserverFacade.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Episerver\Business;
+
+use Generated\Shared\Transfer\MailTransfer;
+use Spryker\Zed\Kernel\Business\AbstractFacade;
+
+/**
+ * @method \SprykerEco\Zed\Episerver\Business\EpiserverBusinessFactory getFactory()
+ */
+class EpiserverFacade extends AbstractFacade implements EpiserverFacadeInterface
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @api
+     *
+     * @param int $idSalesOrder
+     *
+     * @return void
+     */
+    public function mailNewOrderEvent(int $idSalesOrder): void
```

Comment:
In my opinion, this is the correct verb.
We can also use [mail me](https://english.stackexchange.com/a/5031)
Let me know, please, @alex-galych, @gechetspr in case I'm wrong

Reply:
Update: in PHP there is even a method `mail`
http://php.net/manual/en/function.mail.php

Reply:
In my opinion, both options `mail` and `send nail` are correct.

Reply:
@gechetspr are you OK with this?

Reply:
@alex-galych IMO `mail` is not so good as `send`, but this is not a blocker at all